cookbooks/aws-parallelcluster-scheduler-plugin/files/default/event_handler/write_chef_error_handler.rb

```diff
@@ -0,0 +1,9 @@
+module WriteChefError
+  class WriteChefError < Chef::Handler
+    def report
+      if !node['cluster']['chef_error'].nil?
```

Comment:
this still requires that `node['cluster']['chef_error']` is set before reasing the exception. 
Although this works for resources that are under our control, it doesn't work for the other resources/actions for which we don't have control, e.g. third party or built-in. 
I'm wondering if there is a more general way to access the raised exception or message, so that we can use write that message, after an appropriate filtering.